docker-compose/nested-spire/README.md

```diff
@@ -48,15 +48,15 @@ We define all the services for the tutorial in the [docker-compose.yaml](docker-
    services:
      # Root
      root-server:
-       image: gcr.io/spiffe-io/spire-server:1.0.0
+       image: ghcr.io/spiffe/spire-server:1.5.1
        hostname: root-server
        volumes:
          - ./root/server:/opt/spire/conf/server
        command: ["-config", "/opt/spire/conf/server/server.conf"]
      root-agent:
        # Share the host pid namespace so this agent can attest the nested servers
        pid: "host"
-       image: gcr.io/spiffe-io/spire-agent:1.0.0
+       image: ghcr.io/spiffe/spire-agent:1.5.1
```

Comment:
there are some configurations that uses 1.5.1 and another uses 1.5.0, I think we may use 1.5.1 on all places

Reply:
Agreed, updated.